pkg/metrics/cluster/node.go

```diff
@@ -114,15 +115,12 @@ func (n *node) ApplyConfig(cfg Config) error {
 	if err != nil {
 		return fmt.Errorf("failed to create ring: %w", err)
 	}
-	if err := n.reg.Register(r); err != nil {
-		return fmt.Errorf("failed to register ring metrics: %w", err)
-	}
 	if err := services.StartAndAwaitRunning(context.Background(), r); err != nil {
 		return fmt.Errorf("failed to start ring: %w", err)
 	}
 	n.ring = r
 
-	lc, err := ring.NewLifecycler(cfg.Lifecycler, n, "agent", agentKey, false, n.reg)
+	lc, err := ring.NewLifecycler(cfg.Lifecycler, n, "agent", agentKey, false, n.log, n.reg)
```

Comment:
You'll want to wrap the registerer here as well. Since we removed the `cortex_` prefix on the ring metrics in dskit  you'll want to wrap with the `cortex_` prefix here to match the previous cortex metrics.